thirdparty/bullet/BulletDynamics/Featherstone/btMultiBody.cpp

```diff
@@ -343,6 +343,7 @@ void btMultiBody::finalizeMultiDof()
 	m_deltaV.resize(6 + m_dofCount);
 	m_realBuf.resize(6 + m_dofCount + m_dofCount * m_dofCount + 6 + m_dofCount);  //m_dofCount for joint-space vels + m_dofCount^2 for "D" matrices + delta-pos vector (6 base "vels" + joint "vels")
 	m_vectorBuf.resize(2 * m_dofCount);                                           //two 3-vectors (i.e. one six-vector) for each system dof	("h" matrices)
+	m_matrixBuf.resize(m_links.size() + 1);
```

Comment:
Why is this change needed? It should also be PR'ed upstream if it's a bugfix.
Merging that also means that we're no longer compatible with Bullet 2.88 which is the current stable release, so if we can handle this on our end without modifying Bullet for now, that would be best.

Reply:
Nope is not possible to manage it internally, I did a PR on the Bullet side also: https://github.com/bulletphysics/bullet3/pull/2236 but in order to be able to use it ASAP I did it in this way